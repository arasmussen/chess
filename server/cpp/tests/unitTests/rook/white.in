a2:a4
h2:h4
a1:a3
h1:h3
a3:d3
h3:f3
f3:e4
