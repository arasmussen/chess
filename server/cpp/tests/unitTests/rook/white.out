a7:a5
h7:h5
a8:a6
h8:h6
a6:d6
h6:f6
Lose:IllegalMove
